packages/bytes/src/bytes.cairo

```diff
@@ -4,8 +4,8 @@ use alexandria_bytes::utils::{
 use alexandria_math::sha256::sha256;
```

Comment:
should be replaced in favor of syscall

Reply:
here this shouldn't be imported in the `bytes` module. the syscall should be imported here and used for sha256(@Bytes)

Reply:
solved at 846621a1b731590cf316f9bec20e75cfe39b88a4, lmk if good :)